docs/concepts-broker-load-balancing-migration.md

```diff
@@ -0,0 +1,170 @@
+---
+id: concepts-broker-load-balancing-migration
+title: Migration
+sidebar_label: "Migration"
+---
+
+Pulsar has 3 types of broker load balancers, that is, simple, modular, and extensible. 
+
+You can perform the following migrations.
+
+Migration|When to use
```

Comment:
Change "when to use" to "Available Shedding Strategy."

Reply:
"Available Shedding Strategy" is provided below in a table
<img width="485" alt="image" src="https://github.com/apache/pulsar-site/assets/50226895/2978f148-4fe2-4771-b66d-45bca2716d58">

Here want to show "when to use" to give users reminders that they can use a certain type of migration in a certain case